### DISCUSSION TITLE
Summary of tests in Report

### DISCUSSION THREAD

**Discussion by baum-sparton** [ThrowTheSwitch/Ceedling#875] [Q&A] [answered]
Hi, I've been asked to run a regression test on some software.  I have the Ceedling tests and the report from the last time it was ran.  There is a section of the report that I can't figure out how to get generate which seems to provide a sort of description of each test case.  Does anyone recognize a test output that looks like this?  If so any idea how to get it?  Note that these are not comments that appear anywhere in the code.

// Test Description: Tested function prototype for InitS
// 
// Variables tested: Sbits, tSCK, tSDO, tSDI, tEECS
//
// Range:  
//    tSCK = 0;                    //Output for Master
//    tSDO = 0;                    //Output
//    tSDI = 1;                    //Input
//    tEECS =0;                    //Output
//	TEST_ASSERT_EQUAL(1, Sbits.SP);

**Comment by mvandervoord** [comment, marked as answer]
I'm unaware of a plugin which would provide this report. It's a great idea... but not one that I have seen. Perhaps it's a custom plugin that is part of the project you've inherited?

**Reply by baum-sparton** [reply]
Thanks
